test/lib/jdk/test/lib/process/Proc.java

```diff
@@ -256,6 +256,15 @@ public Proc start() throws IOException {
                 }
             }
         }
+        String patchPath = System.getProperty("test.patch.path");
+        if (patchPath != null) {
+            try (var subs = Files.newDirectoryStream(Path.of(patchPath))) {
+                for (var sub : subs) {
+                    var name = sub.getFileName();
+                    cmd.add("--patch-module=" + name + "=" + sub);
+                }
+            }
```

Comment:
Missing catch block?

Reply:
No. I only meant to use try-with-resources, and did not care about the exception.